src/link/stm32_flash_f7_split.ld

```diff
@@ -166,7 +166,7 @@ SECTIONS
 
     . = ALIGN(4);
     _efastram_data = .;        /* define a global symbol at data end */
-  } >FASTRAM AT >AXIM_FLASH
+  } >FASTRAM AT >AXIM_FLASH1
```

Comment:
If we need to reshuffle flash usage, can we please do this in a separate pull request, as this is really unrelated to the proposed change?

Reply:
Actually there are quite a few changes to linker files to tidy up the use of memory suitable for DMA.